manifests/00-crd-operator-config.yaml

```diff
@@ -13,6 +13,32 @@ spec:
   subresources:
     status: {}
   versions:
-  - name: v1
-    served: true
-    storage: true
+    - name: v1
+      served: true
+      storage: true
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            managementState:
+              pattern: ^(Managed|Unmanaged|Removed|Forced)$
+              type: string
+              description: managementState indicates whether and how the operator
+                should manage the component
+            authentication:
+              properties:
+                logoutRedirect:
+                  pattern: ^((https):\/\/?)[^\s()<>]+(?:\([\w\d]+\)|([^[:punct:]\s]|\/?))$
+                  type: string
+                  description: Logout Redirect must be a valid URL
+            customization:
```

Comment:
Also, `customization.documentationBaseURL` and `customization.Brand` should handle `""`/unset. I'm betting these do not. Fast-follow!